chain/client/src/stateless_validation/partial_witness/partial_witness_actor.rs

```diff
@@ -202,11 +202,15 @@ impl PartialWitnessActor {
             main_transition_shard_id,
         } = msg;
 
-        tracing::debug!(
+        let _span = tracing::debug_span!(
             target: "client",
-            chunk_hash=?state_witness.chunk_header().chunk_hash(),
             "distribute_chunk_state_witness",
-        );
+            chunk_hash=?state_witness.chunk_header().chunk_hash(),
+            height_created=?state_witness.chunk_header().height_created(),
```

Comment:
Can you use `height`? it is convenient to have the same name for this attribute across multiple spans.